onnx/src/ops/array/range.rs

```diff
@@ -0,0 +1,104 @@
+use std::ops::{Add, Div, Mul, Sub};
+use tract_ndarray::prelude::*;
+use tract_num_traits::*;
+
+use tract_hir::internal::*;
+
+use crate::model::ParsingContext;
+use crate::pb::*;
+
+#[derive(Debug, Clone, new, Hash)]
+pub struct Range {
+    dtype: DatumType,
```

Comment:
Except on very old TF ops, I don't try and store the operating type except if I need it explicitely. I think you can remove this.